README.md

```diff
@@ -4,21 +4,21 @@
 [![download](/images/icon-google-play.png)]
 (https://play.google.com/store/apps/details?id=io.digibyte)
 
-###DigiByte done right
+### DigiByte done right
 
 
-#####a completely standalone DigiByte wallet
+##### A completely standalone DigiByte wallet
 
 The DigiWallet is a rapidly developing mobile wallet for the Android platform. With extensive work done to enhance sync timing, and innovative features such Digi-ID, the DigiWallet is the premier mobile product of the DigiByte platform.
 
-#####beautiful simplicity:
+##### Beautiful simplicity:
 
-DigiWallet is an extremely simple presenation of how to store, receive, and send DigiByte. As we progress the Beta, currently released on the Google Play Store, new innovative user interface components will breath life into the product, and truly make it stand out amongst the croud.
+DigiWallet is an extremely simple presentation of how to store, receive, and send DigiByte. As we progress the Beta currently released on the Google Play Store, new innovative user interface components will breathe life into the product, and truly make it stand out from the crowd.
```

Comment:
This needs to be re-worded, but I didn't understand the intent of the original sentence, so I don't have a suggestion on how to fix it